.githooks/check-format

```diff
@@ -2,13 +2,13 @@
 
 # Note: This script is intended to be run from the root of the repository.
 #
-# This script checks the format of the code and cmake files.
+# This script checks the format of the C++ code.
```

Comment:
I suppose check-format is no longer a suitable name for this script. Perhaps we should rename it to something like `fix-local-includes`?

Reply:
I didn't do it to show diff properle (not delete+add file).
When I switch to pre-commit hooks only, I will move this file to a separate dir, an example can be found here: https://github.com/XRPLF/clio/pull/2046/files